packages/matter.js/src/protocol/MessageExchange.ts

```diff
@@ -110,6 +117,15 @@ export class MessageExchange<ContextT> {
     private readonly retransmissionRetries: number;
     private readonly messagesQueue = new Queue<Message>();
     private receivedMessageToAck: Message | undefined;
+    private receivedMessageAckTimer = Time.getTimer(MRP_STANDALONE_ACK_TIMEOUT_MS, () => {
+        if (this.receivedMessageToAck !== undefined) {
+            const messageToAck = this.receivedMessageToAck;
+            this.receivedMessageToAck = undefined;
+            this.sendStandaloneAckForMessage(messageToAck).catch(error =>
```

Comment:
We don't have a standard way of tracking "fire-and-forget" operations right now but maybe could add a TODO?

Reasoning is if we shutdown while the ack is outstanding it could crash when it processes, and if this type of operation were o hang they would pile up indefinitely and could cause leaks.

Reply:
TODO added